### PR TITLE
Bug 1883651 - turn off cron and workflows on `main`

### PR DESCRIPTION
**Pull request by jcristau**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1883651

### CODE REVIEW

File: .github/workflows/import-l10n.yml [hunk 1]
```diff
@@ -9,8 +9,6 @@ permissions:
   pull-requests: write
 
 on:
-  schedule:
-    - cron: "0 0 * * *"
```
Comment (flodolo) [inline .github/workflows/import-l10n.yml:13]: We should keep this enabled, otherwise the ac-sync-strings.yml won't have anything to uplift.

Comment (jcristau) [reply .github/workflows/import-l10n.yml]: Thanks for pointing that out!  Updated.

Comment (flodolo) [reply .github/workflows/import-l10n.yml]: There is also a parallel conversation on Slack.

If we want to keep main frozen, we could:
* Disable this workflow (as you did)
* Update the import workflow to write to the release 125 branch. It needs to be tested, but updating the branch reference [here](https://github.com/mozilla-mobile/firefox-android/blob/main/.github/workflows/import-l10n.yml#L25) might be all we need.

**Comment by jcristau** [comment]
Updated to disable the 3 per-product sync-strings workflows, and change the l10n-import workflow to import into the relbranch directly for the 124 cycle.

**Review by flodolo — APPROVED** [review]

**Review by gbrownmozilla — APPROVED** [review]

**Review by gabrielluong — APPROVED** [review]

**Pull request merged**